chia/wallet/puzzles/condition_codes.clvm

```diff
@@ -29,10 +29,10 @@
     ; the conditions below ensure that we're "far enough" in the future
 
     ; wall-clock time
-    (defconstant ASSERT_SECONDS_AGE_EXCEEDS 80)
-    (defconstant ASSERT_SECONDS_NOW_EXCEEDS 81)
+    (defconstant ASSERT_SECONDS_RELATIVE 80)
```

Comment:
When I see `ASSERT_SECONDS_RELATIVE 500` it's a lot less clear than `ASSERT_SECONDS_RELATIVE_EXCEEDS 500`, so I'm not really crazy about the name changes dropping "exceeds". Now there's no verb.

Reply:
That's probably why it was `AGE` & `NOW` before, to keep the name from being too long without losing any semantics.